docs/USAGE.md

```diff
@@ -87,6 +87,8 @@ docker pull ipfs/testground:latest
 
 or
 
+Please, ensure before running the command bellow, make sure you install Docker on your machine 
```

Comment:
That's a good point. `Testground` doesn't really need `Docker`, but if you want to use the `sidecar`, or the docker-related builder or runners, you have to have it.

I suggest we just add Docker as a requirement at the top, alongside Go, and not in the middle of the docs.

Reply:
Also typo: `Please, before running the command below, make sure you install Docker on your machine.`

Reply:
I agree! On it ;)